doc/application/index.rst

```diff
@@ -1446,6 +1446,156 @@ After running the preprocessor, the final device tree used in the
+exist multiple times in the final firmware. This again is useful for
+bootloader images, as they often need to have their own fixed copies
+of the same libraries that the application image is using.
+
+Enabling and configuring multiple images
+========================================
+
+Images are organized in a tree of parent-child relationships. Meaning,
+a parent image needs and enables through Kconfig one or more child
+images. Usually, this tree is branchless, a linked list of images
+where each image enables the child image that boots it. The simplest
+and most common organization is a single application image that
+enables a single bootloader image.
+
+When a parent image enables a child image through Kconfig, both images will be
+configured when CMake is run. During the building of the parent image,
+depending on the configuration of the parent image, the child image will either
+be built from source, merged in as a prebuilt hex file, or ignored. If the
+configuration states that the child image should be included as a hex file or
+built from source, the build system will merge the hex files of the parent and
+child image together so that they can be easily flashed onto the device through
+the "ninja flash" target.  Meaning, for the simplest scenario one can
+enable and integrate an additional image without even realizing it with the
+same workflow as is used for additional libraries, enablement through Kconfig.
+
+Configuring child images is done with the same mechanisms as for the
+parent images. Through CMake variables, Kconfig input fragments, and
+modifying the .config file in the build directory. For instance, to
+one could run ``cmake -Dmcuboot_CONF_FILE=prj_a.conf
+-DCONF_FILE=app_prj.conf``. To change the CONF_FILE for the mcuboot
+image and the parent image. As seen in CONF_FILE, all CMake Cache
+variables that are image-specific are given a prefix to disambiguate
+them. To simplify configuation of single-image builds the toplevel
+image has the empty string as it's prefix.
+
+The same prefix convention is used to disambiguate targets. This means
+that for instance to run menuconfig one would invoke the
+``menuconfig`` target to configure the toplevel image and
+``mcuboot_menuconfig`` to configure the MCUBoot child image.
+
+Defining new child images
+=========================
+
+This section describes how to take an existing parent image and turn
+it into a child image that can be enabled. A parent image typically
+consists of source code, Kconfig fragments, and build script code. The
+source code and Kconfig fragments can be re-used as-is, but several
+changes to the build scripts are necessary.
+
+As mentioned earlier, each target needs a prefix to disambiguate. This
+includes the library target 'app', so any references to 'app' would
+need to be changed to '${IMAGE}app'. Or, even better, one would use
+the zephyr_library_ API instead of the target_* API to indirectly
```

Comment:
As written, this is a trying to create a link to a target called ``zephyr_library`` in this document, but no such target exists (hence the doc build error).  The ``target_*`` is also suspicious.  If you're just wanting to have the text zephyr_library_ show up, you need to escape the final underscore: ``zephyr_library\_`` and the same for ``target\_*``  (Final underscores are the reST notation for an outbound link.)

Reply:
Done

Reply:
I don't see that change applied here.